### PR TITLE
postgresql 9.5.5 (new formula)

### PR DESCRIPTION
**Pull request by whoisjake**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [x] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/master/README.md#acceptable-formulae)?

-----

I couldn't use `brew audit --strict --online postgresql95` as brew audit is failing with a Ruby error. The formula is copied exactly from `homebrew/core`, removing `:head`, adding `revision 1`, and adding a conflict with the base `postgresql` formula. So I would expect this audit to pass.

### CODE REVIEW

File: postgresql95.rb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v9.5.5/postgresql-9.5.5.tar.bz2"
+  sha256 "02c65290be74de6604c3fed87c9fd3e6b32e949f0ab8105a75bd7ed5aa71f394"
+
+  bottle do
+    sha256 "339a45ac04bb079550bbca620a6a7d759e27bfa5203e7bb7f2c9f64b7c2edd59" => :sierra
+    sha256 "59fcad611ef8b50e8ac9194a6f1971b8acb3d0f0ed3a47b9af601caa88d89bdd" => :el_capitan
+    sha256 "44556ea4cb27b61cb931c796c8d5e48298a2b1bedd9830f308bf1e8b8c1cdb23" => :yosemite
+  end
```
Comment (MikeMcQuaid) [inline postgresql95.rb]: Remove the bottle block.

File: postgresql95.rb [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  revision 1
```
Comment (MikeMcQuaid) [inline postgresql95.rb]: Remove the `revision`.

File: postgresql95.rb [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+  option "32-bit"
+  option "without-perl", "Build without Perl support"
+  option "without-tcl", "Build without Tcl support"
+  option "with-dtrace", "Build with DTrace support"
+
+  deprecated_option "no-perl" => "without-perl"
+  deprecated_option "no-tcl" => "without-tcl"
+  deprecated_option "enable-dtrace" => "with-dtrace"
```
Comment (MikeMcQuaid) [inline postgresql95.rb]: Remove all `deprecation_option`s.

File: postgresql95.rb (hunk 1 above)
Comment (whoisjake) [inline postgresql95.rb]: rgr rgr

File: postgresql95.rb (hunk 2 above)
Comment (whoisjake) [inline postgresql95.rb]: rgr rgr

File: postgresql95.rb (hunk 3 above)
Comment (whoisjake) [inline postgresql95.rb]: rgr rgr

**Comment by whoisjake** [comment]
@MikeMcQuaid thank you for the feedback!

**Comment by whoisjake** [comment]
There are some more issues I'm running into the deeper I go. Ack.

**Comment by MikeMcQuaid** [comment]
> I couldn't use brew audit --strict --online postgresql95 as brew audit is failing with a Ruby error.

Can you post details of this here?

Output for initial version:
https://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1726/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_homebrew_versions_postgresql95___new_formula/

**Comment by whoisjake** [comment]
```
$ brew audit --strict --online postgresql95
Error: undefined method `[]' for nil:NilClass
Please report this bug:
  https://git.io/brew-troubleshooting
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:703:in `audit_revision_and_version_scheme'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1071:in `audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:72:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:68:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:68:in `audit'
/usr/local/Homebrew/Library/Homebrew/brew.rb:94:in `<main>'
```

**Comment by whoisjake** [comment]
It's clean now after I made the changes you suggested. 

Right now I'm having problems with initdb on the post install...

```
 ✘ jake /usr/local  $ brew postinstall homebrew/versions/postgresql95
==> /usr/local/Cellar/postgresql95/9.5.5/bin/initdb /usr/local/var/postgres
Last 15 lines from /Users/jake/Library/Logs/Homebrew/postgresql95/post_install.01.initdb:
2016-11-16 10:30:00 -0600

/usr/local/Cellar/postgresql95/9.5.5/bin/initdb
/usr/local/var/postgres

The files belonging to this database system will be owned by user "jake".
This user must also own the server process.

initdb: file "/usr/local/share/postgresql/postgres.bki" does not exist
This might mean you have a corrupted installation or identified
the wrong directory with the invocation option -L.

READ THIS: https://git.io/brew-troubleshooting
```

**Comment by whoisjake** [comment]
Looks like initdb wants:
`/usr/local/share/postgresql/postgres.bki`
but it's really in
`/usr/local/share/postgresql95/postgres.bki`

Not *quite* sure how to fix this yet :(

**Comment by MikeMcQuaid** [comment]
What does your `brew config` say?

**Comment by MikeMcQuaid** [comment]
> Not quite sure how to fix this yet :(

Probably need a `configure` argument.

**Comment by whoisjake** [comment]
Yeah, it looks like some hard coded values for `/postgresql` instead of `/#{name}`

```
$ brew config
HOMEBREW_VERSION: 1.1.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: edf042ee97f80955b077724119e717a51eb25e29
Last commit: 6 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 2ab48fcd080ef7160a28e7b8d8693a87df42b7f7
Core tap last commit: 3 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit ivybridge
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.10.2 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /Users/jake/.rbenv/shims/ruby => /Users/jake/.rbenv/versions/2.3.2/bin/ruby
Java: 1.8.0_101, 1.8.0_66, 1.8.0_45, 1.8.0_20, 1.8.0_05, 1.7.0_79
macOS: 10.12.1-x86_64
Xcode: 8.1
CLT: 8.1.0.0.1.1476494586
X11: 2.7.9 => /opt/X11
```

**Comment by whoisjake** [comment]
Ok, so now it installs clean.

I have a data directory, I can start the service, and I can connect to it.

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by bwlz** [comment]
@MikeMcQuaid can I ask why postgresql93 was removed? It has not yet been EOL'd and my team (and I'm sure many others) are still using it.

**Comment by MikeMcQuaid** [comment]
@bwlz The README explains our guidelines here: https://github.com/Homebrew/homebrew-versions#acceptable-formulae

**Pull request closed without merging**